### PR TITLE
Added getter and setter of workout type for Device class, added const…

### PR DESCRIPTION
**Pull request by ali-akturin**
…ructor for injury, added TODO to Notification class

### CODE REVIEW

File: server/userClasses/injury.py [hunk 1]
```diff
@@ -3,11 +3,16 @@
 
 class Injury:
     NOTIFICATION_TEXT = "This is a reminder to update your injur's pain index."
-    def __init__(self): #TODO 
+    def __init__(self): 
         self.__area = "" #area of the injury (i.e. shoulder, knee)
         self.__pain = 0   #user's pain rating 1-10
         self.__injuryDate = datetime.datetime(0, 0, 0) #the date the injury occured
     
+    def __init__(self, area, pain, injuryDate):
```
Comment (PeterMoraStevens) [inline server/userClasses/injury.py:11]: Do you have two inits?

**Review by PeterMoraStevens — CHANGES_REQUESTED** [review]
Either it's a issue from my view of there are two inits, or the two have a purpose but could you please explain? :)

File: server/userClasses/injury.py (hunk 1 above)
Comment (ali-akturin) [reply server/userClasses/injury.py:11]: yeah, I thought it would be good to have two constructors.

Comment (PeterMoraStevens) [reply server/userClasses/injury.py:11]: Ahhh, makes sense, I understand, thank you for explaining LGTM! :)

**Review by PeterMoraStevens — APPROVED** [review]
Nice update 👍

**Pull request merged**